docs/book/src/developer/guide.md

```diff
@@ -134,6 +152,20 @@ spec:
         name: manager
 ```
 
+```yaml
```

Comment:
I think we can skip this second example given that we are saying

> In both cases, change the `- image:` url to the digest URL mentioned above:

Reply:
Sure will remove this